its/src/test/java/com/sonar/it/scanner/msbuild/CppTest.java

```diff
@@ -65,8 +66,9 @@ public static void checkSkip() {
   @ClassRule
   public static Orchestrator ORCHESTRATOR = Orchestrator.builderEnv()
     .setOrchestratorProperty("cppVersion", "LATEST_RELEASE")
-    .addPlugin("cpp")
-    .activateLicense("cpp")
+    .setEdition(Edition.DEVELOPER)
+    //.addPlugin("cpp")       // FIXME: add cpp plugin
```

Comment:
what about these?

Reply:
Test class is ignored for quite some time so it should be alright not to fix that.